src/handwritten/system/storage/persisted_task.py

```diff
@@ -16,12 +16,14 @@ class PersistedTask:
     """
     __slots__ = ['_name', '__weakref__']
 
-    def __init__(self, name):
+    def __init__(self, name, interpreter):
```

Comment:
This is a public constructor. Adding a required interpreter parameter breaks compatibility with client code that directly constructs it like `nidaqmx.system.storage.PersistedTask("MyTask")`.

See my comments on `scale.py.mako`.